ontology/yaml/resources/SAFETY/entity_types/HDS.yaml

```diff
@@ -17,11 +17,21 @@
 ########################
 
 
+HDS_STD:
+  guid: "05e2fa4b-6ad7-4a63-a8b2-f0cf003dca1e"
+  description: "Standard Hydrogen sensor."
+  implements:
+  - HDS
+  is_canonical: true
+  uses:
+  - particle_concentration_sensor
```

Comment:
Can you use the existing `HDS_H2M` type?

Reply:
yes, I can add the opt_uses: particle_concentration_alarm to HDS_H2M if this is ok?